oshmem/mca/atomic/atomic.h

```diff
@@ -98,6 +98,24 @@ struct mca_atomic_base_module_1_0_0_t {
                         const void *value,
                         size_t nlong,
                         int pe);
+    int (*atomic_fand)(void *target,
```

Comment:
what do you think about separating "op" and "fop"? to avoid extra branch in atomic module functions
so the number of functions here would be 10: add,and,or,xor,fadd,fand,for,fxor,swap,cswap